src/buffer.hh

```diff
@@ -261,14 +265,16 @@ private:
     Flags  m_flags;
 
     using UndoGroup = Vector<Modification, MemoryDomain::BufferMeta>;
+    using WallClock = std::chrono::system_clock;
+    using WallTime = WallClock::time_point;
```

Comment:
Not sure using the system_clock is better than the steady_clock we use by default, as that means we now potentially have children with timestamp earlier than their parent (if the clock was changed in the mean time). I think its safer to just use the existing Clock and TimePoint.